source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm

```diff
@@ -18,7 +18,7 @@ + (CustomInputNumberRenderer *)getInstance
 
 - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
            rootView:(ACRView *)rootView
-             inputs:(NSArray *)inputs
+             inputs:(NSMutableArray *)inputs
```

Comment:
Didnt we make a similar change for 1.2.9? Was it someplace else?

Reply:
No, this is applied to sample app's custom renderer.